test/e2e/specs/specs-playwright/wp-widgets__spec.ts

```diff
@@ -1,7 +1,8 @@
 import { DataHelper, LoginFlow, SidebarComponent, setupHooks } from '@automattic/calypso-e2e';
 import { Page } from 'playwright';
 
-const user = 'gutenbergSimpleSiteUser';
+const user =
+	process.env.GUTENBERG_EDGE === 'true' ? 'gutenbergSimpleSiteEdgeUser' : 'gutenbergSimpleSiteUser';
```

Comment:
I'm pinging @worldomonation to get his take, but my gut tells me here that we should abstract whether we should be testing on Gutenberg Edge to a library function in the `calypso-e2e` package. I know we've be avoiding making too many library functions there, but I think this might a good case. I think a lot of tests will share the Gutenberg Edge concern, and decoupling the specs from a specific ENV name seems like the right idea. That way, that ENV name can change (or we can even move it over to a config-based implementation) without breaking any of the tests.

Not required for this PR, but wanted to toss the suggestion out there!

Reply:
Agreed 👍🏻 . A simple `getGutenbergUser` function that abstracts this would suffice, I think. I can take care of that in a follow-up. Will you create a task and assign it to me (if possible)? Thanks!